### PR TITLE
include docker pulls in total downloads

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,46 +11,57 @@ let total = 0;
...
-for (var version of java_version) {
+function request(url) {
+  const xmlHttp = new XMLHttpRequest();
+  xmlHttp.open("GET", url, false);
+  xmlHttp.send(null);
+  return JSON.parse(xmlHttp.response);
+}
+
+// Dockerhub pulls
+let result = request("https://cors-anywhere.herokuapp.com/https://hub.docker.com/v2/repositories/adoptopenjdk?page_size=100")
```
Comment (karianna) [inline index.js]: What does limiting to 100 mean?

File: index.js [hunk 2]
```diff
@@ -11,46 +11,57 @@ let total = 0;
...
   let item = $(template).clone();
   $(item).find('.count-text').html(version);
   $(item).find('#link').attr("href", "./release.html?version=" + version);
   $(item).css("display", "none");
   $('#versions').append(item);
-  var settings = {
-    "url": `https://api.adoptopenjdk.net/v2/info/releases/${version}`,
-    "method": "GET"
+
+  let result = request(`https://api.adoptopenjdk.net/v2/info/releases/${version}`)
```
Comment (karianna) [inline index.js]: What type of result? More accurate var name might help here.

**Review by karianna — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (gdams) [reply index.js]: the API returns 15 results per page otherwise. 100 is large enough to cover us.

File: index.js (hunk 2 above)
Comment (gdams) [reply index.js]: renamed

**Review by karianna — APPROVED** [review]

**Pull request merged**